tools/DDL/ddl.py

```diff
@@ -144,6 +144,30 @@ def create_demo(demo_id, host, title, state):
         print(f"Could not create demo {demo_id}")
 
 
+def update_ddl(demos):
+    """
+    Update the build.url field in a DDL
+    """
+
+    for editorsdemoid in demos:
+        try:
+            # Open the JSON file and load its content
+            with open("DDLs/" + str(editorsdemoid) + ".json", "r") as file:
+                ddl = json.load(file)
+
+            # Update the build.url field
+            if 'build' in ddl and 'url' in ddl['build']:
+                print(f"Original build URL for {editorsdemoid}: {ddl['build']['url']}")
+                ddl['build']['url'] = f"git@github.com:ipol-journal/{editorsdemoid}.git"
```

Comment:
Maybe check that the previous url was what we expected (git@github.com:mlbriefs/demoid.git), so that we don't override other demos.